specification/metadata/providerprefix.md

```diff
@@ -0,0 +1,17 @@
+# Provider Prefix
```

Comment:
Maybe Provider Column Prefix, to be clear and explicit?

Reply:
Updated